test/e2e/common/runtime.go

```diff
@@ -174,7 +174,12 @@ while true; do sleep 1; done
 				matchTerminationMessage(container, v1.PodSucceeded, Equal("DONE"))
 			})
 
-			It("should report termination message [LinuxOnly] if TerminationMessagePath is set as non-root user and at a non-default path [NodeConformance]", func() {
+			/*
+				Release: v1.14
+				Name: Container Runtime, TerminationMessagePath, non-root user and non-default path
+				Description: Create a pod with an container to run it as non-root user and an custom TerminationMessagePath set. Pod redirect the output to provided path and succeed without failure. When container is terminated, termination message MUST match the expected output recorded in the provided custom path.
+			*/
+			framework.ConformanceIt("should report termination message [LinuxOnly] if TerminationMessagePath is set as non-root user and at a non-default path [NodeConformance]", func() {
```

Comment:
Why exactly is this LinuxOnly?  SecurityContext?

Reply:
Yes, here SecurityContext and RunAsUser are the key reason of [LinuxOnly] tag.

Reply:
Correct, plus it is expecting the /dev/termination-custom-log file to be mounted in the container, which cannot be done ATM with Windows Containers.